non-critical-infra/modules/mailserver/default.nix

```diff
@@ -28,6 +28,9 @@
     ];
   };
 
+  # https://nixos-mailserver.readthedocs.io/en/latest/backup-guide.html
+  services.backup.includes = [ config.mailserver.mailDirectory ];
```

Comment:
That leaves out
- /var/sieve
- /var/dkim
- /var/lib/redis-rspamd

Is that a concious design choice/

Reply:
That covers dkim and sieve. I'm not sure we even train the bayes filter, so we might as well leave it at that.